### PR TITLE
docs: fix links to point to v2 docs

### PR DESCRIPTION
**Pull request by micahborg**
## 📝 Description

This is a patch for v2.
Fixed inline documentation links to point to Chakra UI's v2 site on components and hooks.

### CODE REVIEW

File: website/content/changelog/v1.8.2.mdx [hunk 1]
```diff
@@ -62,8 +62,8 @@ version: 1.8.2
 - [#5527](https://github.com/chakra-ui/chakra-ui/pull/5527)
   [`b6ccf0dd2`](https://github.com/chakra-ui/chakra-ui/commit/b6ccf0dd2e3b72a5b50c9e1c00f1e6e3f3b68690)
   Thanks [@with-heart](https://github.com/with-heart)! - When the
-  [Chakra CLI](https://chakra-ui.com/docs/theming/advanced#theme-typings) fails
-  to generate theme typings, it now exits with a status code of `1`. This
+  [Chakra CLI](https://v2.chakra-ui.com/docs/theming/advanced#theme-typings)
+  fails to generate theme typings, it now exits with a status code of `1`. This
```
Comment (segunadebayo) [inline website/content/changelog/v1.8.2.mdx:66]: Do we really need to update this?

File: website/content/changelog/v1.2.1.mdx [hunk 2]
```diff
@@ -20,7 +20,7 @@ version: 1.2.1
   - Add minor bump for adding subcommand `tokens` to generate Theme Typings
 
     See
-    [https://chakra-ui.com/docs/theming/advanced](https://chakra-ui.com/docs/theming/advanced)
+    [https://v2.chakra-ui.com/docs/theming/advanced](https://v2.chakra-ui.com/docs/theming/advanced)
```
Comment (segunadebayo) [inline website/content/changelog/v1.2.1.mdx:23]: Same for here as well

Comment (micahborg) [reply website/content/changelog/v1.2.1.mdx:23]: I changed all the links that were broken and pointed it to the v2 working link. I would say if the link works either way it shouldn't be changed, but I think the broken links should be changed. I'm open to your reasoning though.

Comment (segunadebayo) [inline website/content/changelog/v1.2.1.mdx:23]:
```suggestion
    [https://v1.chakra-ui.com/docs/theming/advanced](https://v1.chakra-ui.com/docs/theming/advanced)
```

**Pull request merged**